internal/template-validator/tlsinfo/tlsinfo_test.go

```diff
@@ -15,79 +16,200 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
+	ocpconfigv1 "github.com/openshift/api/config/v1"
+
+	"kubevirt.io/ssp-operator/internal/common"
 )
 
+const certSubject = "test.kubevirt.io"
```

Comment:
Can it be moved into the `Describe` below?

Reply:
No, because it is used in function `writeCertificate()` that is outside the `Describe`.